src/effects/VST/VSTEffect.cpp

```diff
@@ -1822,6 +1821,28 @@ void VSTEffect::Unload()
       mAEffect = NULL;
    }
 
+   ResetModuleAndHandle();
```

Comment:
This call within `Unload` is unnecessary.

Reply:
ok, removed